### PR TITLE
translated user guide to english

### PR DESCRIPTION
**Pull request by harshilshah99**
This pull request makes the following changes

- Fixes Issue #6

User Guide in english

### CODE REVIEW

File: docs/en/UsersGuide.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+Accessibility Visualizer is a browser extension that aims to visualize important but visually invisible information to improve the accessibility of web pages.
+It is currently distributed for Google Chrome at [Chrome Web Store](https://chromewebstore.google.com/detail/accessibility-visualizer/idcacekakoknnpbfjcdhnkffgfbddnhk).
+
+## What you can do with Accessibility Visualizer
+
+- Image alt text, heading levels, form labeling, and other situations can be displayed overlays on web pages
+- A warning message will be displayed for any of these that are clearly problematic or should be used with caution.
+- `role="status"` `role="alert"` `role="log"` `aria-live` attribute, to notify visually of changes in the live region created by the `<output>` element To do
+
+Until now, this information has only been available through reading the accessibility tree (Accessibility Object Model) in the browser's developer tools, reading the source code, or actually interacting with it using a screen reader. There was no.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:15]: I think the last "There was no" does not needed

File: docs/en/UsersGuide.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+![Screenshot of extensions menu open](./images/extensions_menu.png)
+
+When you click the Accessibility Visualizer item in the extension menu or the Accessibility Visualizer icon pinned to the toolbar, the Accessibility Visualizer pop-up opens.
+
+![Screenshot of Accessibility Visualizer popup opened](./images/a11y_visualizer_popup.png)
+
+### Tip display
+
+If you check the "Show tip" checkbox in the popup, various information will be displayed as a "tip" on the web page you are viewing.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:35]: In English environments, it shows as "Show tips".
(I am not sure if this is natural English expression, but the notation should be same between the document and the software.)

File: docs/en/UsersGuide.md (hunk 1 above)
Comment (ymrl) [reply docs/en/UsersGuide.md:15]:
```suggestion
Until now, this information has only been available through reading the accessibility tree (Accessibility Object Model) in the browser's developer tools, reading the source code, or actually interacting with it using a screen reader.

```

File: docs/en/UsersGuide.md (hunk 2 above)
Comment (ymrl) [reply docs/en/UsersGuide.md:35]:
```suggestion
If you check the "Show tips" checkbox in the popup, various information will be displayed as a "tip" on the web page you are viewing.

```

File: docs/en/UsersGuide.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+!["Komaru City" logo and the heading "Global Warming Prevention Division." The logo is surrounded by a green dotted line and covered by a red chip that says ``Image without alt attribute.'' The logo and headline are surrounded by a blue dotted line, and above them are a blue tip for ``Heading level 1'' and a green tip for ``Global Warming Prevention Division''](./images/tip_example_komaru_city.png )
+
+(Screenshots are from [Komaru City - Experience site for accessibility issues -] (https://a11yc.com/city-komaru/). /city-komaru/practice/?preset=ng-terrible1&wcagver=22)
```
Comment (ymrl) [inline docs/en/UsersGuide.md:39]: The link is broken, and the original web site doesn't seem to have English version, It should written in original language.

```suggestion
(Screenshots is from [たいへんな駒瑠市](https://a11yc.com/city-komaru/practice/?preset=ng-terrible1&wcagver=22) of [駒瑠市〜アクセシビリティ上の問題の体験サイト〜](https://a11yc.com/city-komaru/). /city-komaru/practice/?preset=ng-terrible1&wcagver=22)
```

File: docs/en/UsersGuide.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+You can change the target for which tips are displayed using the checkboxes in the popup. If you want to check a specific target in detail, or if you find it difficult to browse the web if it is left open, uncheck it and make adjustments.
+
+- image
+- button
+- Link
+- form controls
+- Heading
+- aria-hidden
+
+The "Chip opacity" slider allows you to adjust the color intensity of the chip display. If the display is distracting, lower the opacity.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:50]:
```suggestion
The "Tips opacity" slider allows you to adjust the color intensity of the chip display. If the display is distracting, lower the opacity.
```

File: docs/en/UsersGuide.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+## Live region announcement
+
+When "Announce live region" was checked in the popup, there was a change in [ARIA live regions](https://developer.mozilla.org/ja/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) The content will now be displayed near the center of the screen.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:54]: I found English version of this page, it is more suitable.

```suggestion
When "Announce live regions" was checked in the popup, there was a change in [ARIA live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) The content will now be displayed near the center of the screen.
```

File: docs/en/UsersGuide.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+This is a function that allows users of assistive technology such as screen readers to experience and check the operation of the live region function, which is used to inform users of changes in screen status, etc., without using screen readers. .
+
+![A large message "Download now! Speed up your PC!" is displayed in the center of the browser](./images/a11y_visualizer_live_region.png)
+
+(The screenshot is from [aria-live is noisy (ARIA-Barriers)](https://shuaruta.github.io/ARIA-Barriers/2023/12/22/aria-live.html), which displays the announcement. )
```
Comment (ymrl) [inline docs/en/UsersGuide.md:59]:
```suggestion
(The screenshot is from ["aria-live がうるさい (aria-live is noisy)" (ARIA-Barriers)](https://shuaruta.github.io/ARIA-Barriers/2023/12/22/aria-live.html), which displays the announcement. )

```

File: docs/en/UsersGuide.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+## Usage notes
+
+- The display position of the chip may sometimes shift. In that case, please press the "Refresh" button in the popup
+- There are some websites that slow down when displaying tips or announcements. When viewing such sites, please uncheck "Show tip" and "Announce live region".
+- For some sites that are unavoidably slow, extension developers may take measures to prevent them from working.
+- Live region announcements may not be possible due to technical limitations where frames are used.
+
+## Chip display details
+
+Here we briefly explain the points you should check regarding the information displayed on the chip.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:74]:
```suggestion
Here we briefly explain the points you should check regarding the information displayed on the tip.

```

File: docs/en/UsersGuide.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+
+There are following types of chips
```
Comment (ymrl) [inline docs/en/UsersGuide.md:76]:
```suggestion
There are following types of tips

```

File: docs/en/UsersGuide.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+- Name: Appears in green with a humanoid icon. Please check that the content is appropriate and that there are no omissions.
+- Description: Appears in gray with a document icon. Please check that the content is appropriate and that there are no omissions.
+- Heading: Displayed in blue with a bookmark icon. Check if the heading level is appropriate
+- Warning: Displayed in yellow with a warning triangle icon. Indicates where there may be a problem
+- Error: Displayed in red with an error triangle icon. Definitely indicates where the problem is
+- Role: pink and displayed with a tag icon
+- Element: brown and displayed with a `</>` icon that mimics an HTML tag
+
+"Name" is "[Accessible Name](https://developer.mozilla.org/ja/docs/Glossary/Accessible_name)" and "Description" is "[Accessible Description]( https://developer.mozilla.org/ja/docs/Glossary/Accessible_description)" value is displayed. These are pieces of information that help users of assistive technologies, such as screen readers, recognize the element.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:86]:
```suggestion
"Name" is "[Accessible Name](https://developer.mozilla.org/en-US/docs/Glossary/Accessible_name)" and "Description" is "[Accessible Description](https://developer.mozilla.org/en-US/docs/Glossary/Accessible_description)" value is displayed. These are pieces of information that help users of assistive technologies, such as screen readers, recognize the element.

```

File: docs/en/UsersGuide.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+
+### image
+
+When "Show image information" is checked, tips are displayed for `<img>` elements, `<svg>` elements, and elements with the `role="img"` attribute.
+
+- So-called alternative text (alt text) is displayed in the name chip
```
Comment (ymrl) [inline docs/en/UsersGuide.md:92]:
```suggestion
- So-called alternative text (alt text) is displayed in the name tip
```

File: docs/en/UsersGuide.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+   - Image alt text should be a concise description that conveys much of the same information even if the image were displayed instead.
+   - For `<img>` elements, the `alt` attribute is usually used
+   - For `<svg>` element, `<title>` element, `aria-label` attribute, `aria-labelledby` attribute may be used
+   - For elements with `role="img"` attribute, `aria-label` attribute, `aria-labelledby` attribute may be used
+- If `alt=""` is specified in the `<img>` element, a warning tip "Image with alt="" will be displayed. Images in this state cannot be perceived by assistive technologies such as screen readers. **Please provide alt text unless this image is placed for decorative purposes**
+- If no alternative text is specified and it is not `aria-hidden` or `alt=""`, the error tip ``Image without alt attribute'' or ``No name (label)'' will be displayed. . In this case, **Modification is required**
+
+### button
+
+When "Show button information" is checked, `<button>` element, `<input>` element with `type` attribute of `button` `submit` `reset` `image`, `role= Display a tip for elements with the "button" attribute.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:102]:
```suggestion
When "Show about buttons" is checked, tips are displayed for`<button>` element, `<input>` element with `type` attribute of `button` `submit` `reset` `image`, and any elements with `role="button"` attribute. 
```

File: docs/en/UsersGuide.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+
+- The name chip displays the button label. Please check if there are any omissions or if the labels are appropriate.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:104]:
```suggestion
- The name tip displays the button label. Please check if there are any omissions or if the labels are appropriate.

```

File: docs/en/UsersGuide.md [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+- Warning: Displayed in yellow with a warning triangle icon. Indicates where there may be a problem
+- Error: Displayed in red with an error triangle icon. Definitely indicates where the problem is
+- Role: pink and displayed with a tag icon
+- Element: brown and displayed with a `</>` icon that mimics an HTML tag
+
+"Name" is "[Accessible Name](https://developer.mozilla.org/ja/docs/Glossary/Accessible_name)" and "Description" is "[Accessible Description]( https://developer.mozilla.org/ja/docs/Glossary/Accessible_description)" value is displayed. These are pieces of information that help users of assistive technologies, such as screen readers, recognize the element.
+
+### image
+
+When "Show image information" is checked, tips are displayed for `<img>` elements, `<svg>` elements, and elements with the `role="img"` attribute.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:90]:
```suggestion
When "Show about iamge" is checked, tips are displayed for `<img>` elements, `<svg>` elements, and elements with the `role="img"` attribute.
```

File: docs/en/UsersGuide.md [hunk 14]
```diff
@@ -0,0 +1,140 @@
...
+
+When "Show button information" is checked, `<button>` element, `<input>` element with `type` attribute of `button` `submit` `reset` `image`, `role= Display a tip for elements with the "button" attribute.
+
+- The name chip displays the button label. Please check if there are any omissions or if the labels are appropriate.
+- If no name is given, a **``No name (label)'' error tip** will be displayed. In this case, assistive technologies such as screen readers cannot predict the behavior of the button. **Modification required**
+- If the element has the `role="button"` attribute and is not focusable by default, and the `tabindex` attribute is not specified, a **``Unfocusable'' error tip** will be displayed. In this state, you cannot operate with the keyboard, so **Modification is required**
+
+### Link
+
+When "Show link information" is checked, tips are displayed for `<a>` elements, `<area>` elements, and elements with the `role="link"` attribute.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:110]:
```suggestion
When "Show about links" is checked, tips are displayed for `<a>` elements, `<area>` elements, and elements with the `role="link"` attribute.
```

File: docs/en/UsersGuide.md [hunk 15]
```diff
@@ -0,0 +1,140 @@
...
+
+- The text of the link will be displayed in the name tip. Please check if there are any omissions or if the text is appropriate.
+- If the link name becomes empty, a **"No name (label)" error tip** will be displayed. In this case, assistive technologies such as screen readers cannot perceive the purpose of the link. **Modification required**
+- If the `<a>` or `<area>` element does not have the `href` attribute, the browser will not treat it as a link. You will see a warning chip with "No href attribute". If an interaction such as clicking is set for the `<a>` element in this state, keyboard operations may not be possible, or users of assistive technology such as screen readers may not be able to recognize that it is the target of the operation. **In that case, modification is required**
+
+### Form controls
+
+When "Show form control information" is checked, `<input>` elements whose `type` attribute is `hidden` `button` `submit` `reset` `image`, `<textarea>` Displays a tip for elements, `<select>` elements, and elements whose `role` attribute specifies `textbox`, `combobox`, `checkbox`, `radio`, or `switch`.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:118]:
```suggestion
When "Show about form-controls" is checked, tips are displayed for `<input>` elements whose `type` attribute is `hidden` `button` `submit` `reset` `image`, `<textarea>` elements, `<select>` elements, and elements whose `role` attribute specifies `textbox`, `combobox`, `checkbox`, `radio`, or `switch`.

```

File: docs/en/UsersGuide.md [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+
+- Name tip displays form control labels. Please check if there are any omissions or if the labels are appropriate.
+   - Typically, the `<label>` element is used for the `<input>` `<select>` `<textarea>` element
+- If no name is given, a **``No name (label)'' error tip** will be displayed. In this case, assistive technologies such as screen readers cannot perceive the purpose of the form control. **Modification required**
+- If the element is not focusable by default and the `tabindex` attribute is not specified, a **``Unfocusable'' error tip** will be displayed. In this state, you cannot operate with the keyboard, so **Modification is required**
+
+### Heading
+
+When "Show heading information" is checked, tips are displayed for elements `<h1>` to `<h6>` and elements with the `role="heading"` attribute.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:127]:
```suggestion
When "Show about headings" is checked, tips are displayed for elements `<h1>` to `<h6>` and elements with the `role="heading"` attribute.
```

File: docs/en/UsersGuide.md [hunk 17]
```diff
@@ -0,0 +1,140 @@
...
+- The heading level tip displays the heading level. Please make sure it is at the appropriate level.
+   - `<h1>` to `<h6>` elements are used to indicate heading levels
+   - Elements with the `role="heading"` attribute use the `aria-level` attribute to indicate the heading level
+- If an element has the `role="heading"` attribute and the `aria-level` attribute does not exist, a **``No heading level'' error tip** will be displayed. In this case, assistive technologies such as screen readers cannot perceive the level of the heading. **Modification required**
+- The name tip displays the heading text. Please check if there are any omissions or if the text is appropriate.
+- If a heading is not given a name, a ``No name (label)'' error tip will be displayed. In this case, assistive technologies such as screen readers will not be able to perceive the heading. **Modification required**
+
+### aria-hidden
+
+When "Show aria-hidden information" is checked, a tip is displayed for elements with the `aria-hidden="true"` attribute.
```
Comment (ymrl) [inline docs/en/UsersGuide.md:138]:
```suggestion
When "Show about aria-hidden" is checked, tips are displayed for elements with the `aria-hidden="true"` attribute.
```

File: docs/en/UsersGuide.md [hunk 18]
```diff
@@ -0,0 +1,140 @@
...
+
+- A warning tip indicates that the `aria-hidden="true"` attribute is present. Elements with this attribute are invisible to users of assistive technologies such as screen readers. If there is information displayed on the screen that the user can perceive, but `aria-hidden="ture"` is specified, only users of assistive technology such as screen readers will not be able to perceive that information. Masu. If there is an element that is visually visible but marked as `aria-hidden` except for decorative elements, **needs modification**
```
Comment (ymrl) [inline docs/en/UsersGuide.md:140]:
```suggestion
- A warning tip indicates that the `aria-hidden="true"` attribute is present. Elements with this attribute are invisible to users of assistive technologies such as screen readers. If there is information displayed on the screen that the user can perceive, but `aria-hidden="true"` is specified, only users of assistive technology such as screen readers will not be able to perceive that information. If there is an element that is visually visible but marked as `aria-hidden` except for decorative elements, **needs modification**
```

**Review by ymrl — CHANGES_REQUESTED** [review]
Thank you translating my document, it is really helpful!!
I read your translation, and commented some suggestions.

Please commit the suggestions or more better fixes for them, Then I will merge this Pull Request.

**Comment by harshilshah99** [comment]
I have commited for the above suggestions, please check it

**Review by ymrl — APPROVED** [review]
Thank you, Great Job.
ありがとうございます

**Pull request merged**